### PR TITLE
added unicode to ascii conversion

### PR DESCRIPTION
**Pull request by ManjotS**
also fixed some lint issues

### CODE REVIEW

File: src/alphabetize.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const customSort = (firstEl, secondEl) => {
   return firstIndex > secondIndex;
 };
 
-const getKeyByValue = (object, value) => Object.keys(object).find((key) => object[key] === value);
+const getKeyByValue = (object, value) => Object.keys(object).find(key => object[key] === value);
```
Comment (tsingh777) [inline src/alphabetize.js]: please remove this change to fix lint

File: src/alphabetize.js [hunk 2]
```diff
@@ -110,13 +110,13 @@ function alphabetize(sentenceArray, type = 'english') {
     matraFixes.forEach((e) => {
       newSentence = sentence.replace(new RegExp(e[0], 'g'), e[1]);
     });
-    const arr = newSentence.split('').filter((a) => sortedValues.indexOf(a) > 0);
+    const arr = newSentence.split('').filter(a => sortedValues.indexOf(a) > 0);
```
Comment (tsingh777) [inline src/alphabetize.js]: please remove this change to fix lint

File: src/alphabetize.js [hunk 3]
```diff
@@ -110,13 +110,13 @@ function alphabetize(sentenceArray, type = 'english') {
...
     sentenceObj[sentence] = arr.join('');
   });
 
   const sortedResult = Object.values(sentenceObj).sort(customSort);
 
-  return sortedResult.map((value) => getKeyByValue(sentenceObj, value));
+  return sortedResult.map(value => getKeyByValue(sentenceObj, value));
```
Comment (tsingh777) [inline src/alphabetize.js]: please remove this change to fix lint

File: src/unicode.js [hunk 4]
```diff
@@ -120,22 +226,108 @@ const halfChars = [
...
+ * @param {string} unicode text script to be converted
+ * @returns {string} Returns ascii text
+ * @example
+ *
+ * unicode('ਆਇ ਮਿਲੁ ਗੁਰਸਿਖ ਆਇ ਮਿਲੁ ਤੂ ਮੇਰੇ ਗੁਰੂ ਕੇ ਪਿਆਰੇ ॥');
+ * // => 'Awie imlu gurisK Awie imlu qU myry gurU ky ipAwry ]'
+ */
+
+function ascii(text = '') {
+  let convertedText = [];
```
Comment (tsingh777) [inline src/unicode.js]: make this `const` to fix lint.

**Review by tsingh777 — CHANGES_REQUESTED** [review]

File: src/alphabetize.js (hunk 1 above)
Comment (ManjotS) [reply src/alphabetize.js]: lint told me to do this..

File: src/unicode.js [hunk 5]
```diff
@@ -120,22 +231,114 @@ const halfChars = [
...
+  return convertedText.join('');
+}
+
 /**
  * Convert Gurmukhi script to Unicode
  *
  * @since 1.0.0
- * @param {string} text Gurbani Akhar script to be converted
+ * @param {string} text Gurbani Akhar or unicode script to be converted
+ * @param {boolean} do a unicode to ascii (true) or not (false)
```
Comment (tsingh777) [inline src/unicode.js:368]: can you add an example here and to the `README.md` as well.

File: src/unicode.js [hunk 6]
```diff
@@ -120,22 +231,114 @@ const halfChars = [
...
  */
 
-function unicode(text = '') {
+function unicode(text = '', reverse = false) {
   if (text === '' || typeof text !== 'string') {
     return text;
   }
+
+  if (reverse) {
+    return ascii(text);
```
Comment (tsingh777) [inline src/unicode.js:385]: Any reason not make this its own module?

Comment (ManjotS) [reply src/unicode.js:385]: theres already a module called ascii that does something else so it seemed better to just add a "reverse" flag.

File: src/unicode.js (hunk 5 above)
Comment (ManjotS) [reply src/unicode.js:368]: example is below your comment

Comment (tsingh777) [reply src/unicode.js:368]: An example where reverse is passed as true?

line 249 has it but without the second parmeter. 
```
unicode('ਆਇ ਮਿਲੁ ਗੁਰਸਿਖ ਆਇ ਮਿਲੁ ਤੂ ਮੇਰੇ ਗੁਰੂ ਕੇ ਪਿਆਰੇ ॥');
 * // => 'Awie imlu gurisK Awie imlu qU myry gurU ky ipAwry ]'
```
can we add it to README?

Comment (ManjotS) [reply src/unicode.js:368]: done

**Review by tarunsingh5 — DISMISSED** [review]

**Review by tsingh777 — DISMISSED** [review]

**Review by tsingh777 — APPROVED** [review]
code LGTM

**Pull request merged**